File: 3110-score-of-a-string/3110-score-of-a-string.py
class Solution(object):
    def scoreOfString(self, s):
        leng = len(s)
        i = 0
        j = 1
        res = 0
        while i < leng and j < leng:
            res += abs(ord(s[i]) - ord(s[j]))
            i += 1   
            j += 1
        return res
            
        
        

File: 3110-score-of-a-string/3110-score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
        int i = 0;
        int j = 1;
        int len = s.length();
        int res = 0;
        while (i < len && j < len)
        {
            res += Math.abs((int)s.charAt(i) - (int)s.charAt(j));
            i ++;
            j ++;
        }
        return res;
    }
}